CHANGELOG.md

```diff
@@ -2,6 +2,32 @@
 
 All notable changes to this project will be documented in this file.
 
+## [v5.5.0](https://github.com/voxpupuli/puppetboard/tree/v5.5.0) (2024-11-24)
+
+[Full Changelog](https://github.com/voxpupuli/puppetboard/compare/v5.4.0...v5.5.0)
+
+**Features**
+
+- Add support for Python 3.12, 3.13
+- Drop support for Python 3.8
```

Comment:
Isn't this typically a backwards incompatible change?

Reply:
It is, the PR (#1184) is properly tagged, my guess is that this has not been generated by github-changelog-generator.

@d1nuc0m can you check you generated with something like `bundle exec rake changelog`?

Reply:
Nevermind, found [RELEASE.MD](https://github.com/voxpupuli/puppetboard/blob/master/RELEASE.md)